CHANGELOG.md

```diff
@@ -15,6 +15,12 @@ it according to semantic versioning. For example, if your PR adds a breaking cha
 should change the heading of the (upcoming) version to include a major version bump.
 
 -->
+# 5.13.4
```

Comment:
Can you update to `5.14.0` since we are adding a new feature!

Reply:
Most definitely, there you go